### PR TITLE
Add instructions for connecting to VNC session

### LINKED ISSUE #15: Add instructions on VNC

### PR DESCRIPTION
**Pull request by candleindark**
This PR add instructions for connecting to VNC session. It closes #15.

### CODE REVIEW

File: conboarding.md [hunk 1]
```diff
@@ -116,8 +116,48 @@ How much does it cost to run things?
       and [John Hudson's course notes](https://rcweb.dartmouth.edu/~john/HPC/).
    4. Request `rc-DBIC` group from Yarik. (Necessary to use Datalad) Be sure to follow this guide to use DBIC-installed git-annex. https://dbic-handbook.readthedocs.io/en/latest/mri/dataaccess.html#discovery-filesystem
 
-3. ReproNim: request iam from David for AWS Access
-4. DANDI: request credentials for DANDI from Satra
+3. VNC is available on `typhon`
```
Comment (yarikoptic) [inline conboarding.md]: just replace `typhon` with some `SERVERNAME` ;-)  no need to disclose such things too specifically and instructions should generalize for any SERVERNAME really .

Comment (candleindark) [reply conboarding.md]: done

File: conboarding.md [hunk 2]
```diff
@@ -116,8 +116,51 @@ How much does it cost to run things?
...
+
+      ```xtightvncviewer -via <username>@<server name>.dartmouth.edu localhost:<VNC assgined port number>```
+
+      If you are using a VNC client that does not have builtin SSH tunneling, you will
+      need to create an SSH tunnel to the VNC server and then connect to the VNC session
+      through the tunnel.
+
+      1. Create an SSH tunnel to the VNC server
+
+         ```ssh -L <local port number>:localhost:<VNC assgined port number> <username>@<server name>.dartmouth.edu```
```
Comment (yarikoptic) [inline conboarding.md]:
```suggestion
         ```ssh -L <local port number>:localhost:<VNC assigned port number> <username>@<server name>.dartmouth.edu```
```

**Comment by yarikoptic** [comment]
Thank you!

**Pull request merged**